src/components/ContactGroups/ContactGroupManager.jsx

```diff
@@ -1,46 +1,62 @@
-import React from 'react'
+import React, { Component } from 'react'
 import PropTypes from 'prop-types'
-import { Button } from 'cozy-ui/react'
+import { Button, Overlay } from 'cozy-ui/react'
 import { translate } from 'cozy-ui/react/I18n'
-import SelectBox, {
-  reactSelectControl,
-  CheckboxOption
-} from 'cozy-ui/react/SelectBox'
+
+import classNames from 'classnames'
+
+import SelectBox, { CheckboxOption, components } from 'cozy-ui/react/SelectBox'
 import Icon from 'cozy-ui/react/Icon'
 import palette from 'cozy-ui/stylus/settings/palette.json'
 
 import IconGroups from '../../assets/icons/groups.svg'
 import IconDown from '../../assets/icons/down.svg'
+import ContactGroupCreation from './ContactGroupCreation'
 
-const MainButton = ({ t }) => (
-  <Button
-    theme="secondary"
-    size="small"
-    icon={<Icon icon={IconGroups} color={palette['coolGrey']} />}
-    label={t('groups.manage')}
-  >
-    <Icon
-      icon={IconDown}
-      color={palette['coolGrey']}
-      width="12"
-      className="group-manager__indicator"
-    />
-  </Button>
-)
+const MainButton = ({ t, selectProps: { toggleMenuIsOpen } }) => {
+  return (
+    <Button
+      theme="secondary"
+      size="small"
+      icon={<Icon icon={IconGroups} color={palette['coolGrey']} />}
+      label={t('groups.manage')}
+      onClick={() => {
+        toggleMenuIsOpen()
+      }}
+    >
+      <Icon
+        icon={IconDown}
+        color={palette['coolGrey']}
+        width="12"
+        className="group-manager__indicator"
+      />
+    </Button>
+  )
+}
 
 MainButton.propTypes = {
   t: PropTypes.func.isRequired
 }
 
 const MainButtonWithTranslation = translate()(MainButton)
 
-const MainButtonControl = reactSelectControl(<MainButtonWithTranslation />)
-
+const MainButtonControl = props => <MainButtonWithTranslation {...props} />
 const customStyles = {
   container: base => ({
     ...base,
     display: 'inline-block',
-    verticalAlign: 'middle'
+    verticalAlign: 'middle',
+    zIndex: 100000
```

Comment:
I suggest we make the following change:

```
const zIndexAboveOverlay = 51 // see https://github.com/cozy/cozy-ui/blob/master/stylus/settings/z-index.styl#L42
…
verticalAlign: 'middle',
    zIndex: zIndexAboveOverlay
```

So we know why we have a z-index here, and it also prevents going going over things we don't want.